src/content/en/fundamentals/primers/shadowdom/index.markdown

```diff
@@ -680,9 +678,15 @@ The events that **do** cross the shadow boundary are:
 - Keyboard Events: keydown, keyup
 - Composition Events: compositionstart, compositionupdate, compositionend
 
+**Tips**
+
 If the shadow tree is open, calling `event.composedPath()` will return an array of nodes that the event traveled through.
-{: .wf-talkinghead }
 
+If you want to fire custom events from a custom element that uses shadow DOM, be sure to set the `composed` flag when creating the event:
+
+    this.dispatchEvent(new Event('special-thing', {bubbles: true, composed: true});
+
+If `composed: false` (default), consumers won't be able to listen for the event outside of your shadow root.
```

Comment:
Hmm, is this true? I thought it would just retarget it to your shadow host. Which is probably what you want to do in most cases, so suggesting that people always set `composed: true` for firing custom events inside a shadow DOM seems wrong.